packages/malloy/src/lang/grammar/Malloy.g4

```diff
@@ -543,6 +544,7 @@ WEEK: W E E K S?;
 WHEN: W H E N ;
 WITH: W I T H ;
 YEAR: Y E A R S?;
+UNGROUP: U N G R O U P;
```

Comment:
I talked to lloyd a little and I think I like `ungrouped()` better than `ungroup()` because it is more like an adverb than a verb

Reply:
Works for me.